Demo/src/main/java/com/paypal/android/ui/paypalweb/PayPalWebViewModel.kt

```diff
@@ -173,6 +154,25 @@ class PayPalWebViewModel @Inject constructor(
     }
 
     fun handleBrowserSwitchResult(activity: ComponentActivity) {
-        authState?.let { paypalClient?.completeAuthChallenge(activity.intent, it) }
+        val result = authState?.let { paypalClient?.finishStart(activity.intent, it) }
```

Comment:
I had to do a double take on this because our checkout main function is "start" 
But it goes with the pattern. finish<function>

Reply:
Yeah honestly I wouldn't mind renaming it to either `checkout()` or `confirmPaymentSource()`. It might be nice to have `CardClient.confirmPaymentSource()` as well instead of `approveOrder()` to align with PPaaS.

Reply:
Then it would be `finishCheckout()` or `finishConfirmPaymentSource()`.

Reply:
My bad I missed this. I agree we should hold off on re-naming until we're closer to GA.